proprietary/tokens/src/brand/ams/color.tokens.json

```diff
@@ -1,21 +1,85 @@
 {
   "ams": {
     "color": {
-      "primary-black": { "value": "#000000" },
-      "primary-white": { "value": "#FFFFFF" },
-      "primary-blue": { "value": "#004699" },
-      "primary-red": { "value": "#EC0000" },
-      "dark-blue": { "value": "#102E62" },
-      "orange": { "value": "#FF9100" },
-      "yellow": { "value": "#FFE600" },
-      "green": { "value": "#BED200" },
-      "dark-green": { "value": "#00A03C" },
-      "blue": { "value": "#009DE6" },
-      "purple": { "value": "#A00078" },
-      "magenta": { "value": "#E50082" },
-      "neutral-grey1": { "value": "#E8E8E8" },
-      "neutral-grey2": { "value": "#BEBEBE" },
-      "neutral-grey3": { "value": "#767676" }
+      "border": {
+        "default": { "value": "#181818" }
+      },
+      "canvas": {
+        "default": { "value": "#ffffff" }
+      },
+      "feedback": {
+        "error": {
+          "default": { "value": "#ec0000" }
+        },
+        "info": {
+          "default": { "value": "#009de6" }
+        },
+        "success": {
+          "default": { "value": "#00a03c" }
+        },
+        "warning": {
+          "default": { "value": "#ff9100" }
+        }
+      },
+      "footer": {
+        "default": { "value": "#004699" }
+      },
```

Comment:
I was thinking about these one-offs. If the purpose of these tokens is to tell developers: 'These are the colors you can use and when to use them', these one-offs are a little odd. Developers are never supposed to use them, right?

We could just define the color code in the Footer component tokens, seeing as they're not really reusable tokens.
Same goes for the Switch and Logo tokens.

Reply:
Yes, that does sound logical. I’ve removed footer, logo, and switch track.

As we considered, I’ve moved ‘canvas’ into a new ‘background’ group and split off a background colour for inputs. I spent time finding a good structural name for it – we shouldn’t use component or group names in ‘brand’ – and came up with things like ‘notch’, ‘indent’, ‘hole’, and eventually ‘opening’. These components carve out an area from the canvas. Some components that set a white background colour (e.g. secondary button) now use an RGB value again, as they neither indicate a canvas or an opening. We could try moving this colour into ‘interactive’ as well, but it’ll be a bit far-fetched.

That only leaves us with the separator colour, which now forms its own group.

Reply:
> These components carve out an area from what’s on top of the canvas.

Re-reading this makes me think we can, or even should, use the ‘canvas’ colour for inputs if they are openings that make the underlying colour appear.